rest/data/commit_queue.go

```diff
@@ -318,6 +318,28 @@ func EnqueuePRToCommitQueue(ctx context.Context, env evergreen.Environment, sc C
 		return nil, errors.New("PR contains no base branch label")
 	}
 
+	if !thirdparty.IsUnblockedGithubStatus(*pr.MergeableState) {
+		errMsg := fmt.Sprintf("PR is not mergeable, status: %s", *pr.MergeableState)
+		grip.Error(message.Fields{
+			"message":        errMsg,
+			"state":          *pr.MergeableState,
+			"owner":          userRepo.Owner,
+			"repo":           userRepo.Repo,
+			"pr_title":       *pr.Title,
+			"commit_message": *pr.Number,
+		})
+		sendErr := thirdparty.SendCommitQueueGithubStatus(env, pr, message.GithubStateFailure, errMsg, "")
+
+		grip.Error(message.WrapError(sendErr, message.Fields{
```

Comment:
I will add an alert for when we see a blocked PR whose status is not "blocked" so that we can assess it and add it to the list so that we can let it through.

Reply:
Do you mean if the state isn't one of the ones we have in the list but also isn't equal to blocked? I'm not sure I see where we have the information for this alert.

Reply:
my bad, I meant to add this comment to line 323.

Reply:
yes, if something is not equal to blocked but we are blocking on it, we should investigate what the status is to see if we want to add it to the list of statuses to let through, or if "blocked" was changed to something else.

Reply:
That makes sense. Can we update the rotation playbook to handle this alert?